src/__tests__/octokit.test.ts

```diff
@@ -86,32 +95,55 @@ describe("octokitApp", () => {
 });
 
 describe("getOctokit", () => {
+  let privateKey: string;
+
+  beforeAll(() => {
+    const keys = generateKeyPairSync("rsa", {
+      modulusLength: 2048,
+      publicKeyEncoding: { type: "spki", format: "pem" },
+      privateKeyEncoding: { type: "pkcs8", format: "pem" },
+    });
+    privateKey = keys.privateKey;
+  });
   it("throws error when installation id is not a number", async () => {
     const inputInstallationId = "installation_id";
 
     await expect(
-      getOctokit("private_key", "456", inputInstallationId)
+      getOctokit(
+        privateKey,
+        "456",
+        inputInstallationId,
+        {},
+        (parameter: string) => Promise.resolve(parameter)
+      )
+    ).rejects.toThrow("installation_id is not a number");
+  });
+  it("sets empty values on the octokit app when cannot find the parameters", async () => {
+    await expect(
+      getOctokit("private_key", "app_id", "installation_id", {}, () =>
+        Promise.resolve(null)
+      )
```

Comment:
```suggestion
      getOctokit("private_key", "app_id", "installation_id", {}, async () => null);
```

Marginally cleaner, I think

Reply:
Line wrapping is a pain in this case but I'll live with it.